tools/make/data.toml

```diff
@@ -168,24 +168,17 @@ if greater_than ${output_length} 0
     msg = array_join ${msg} "\n"
     trigger_error ${msg}
 end
-
-exec target/debug/icu4x-datagen --locales basic moderate --keys experimental-all --format blob --out /dev/null --overwrite
 '''
 
 [tasks.full-data]
 description = "Builds a complete postcard file from latest data."
 category = "ICU4X Data"
-command = "cargo"
-args = [
-    "run",
-    "--bin=icu4x-datagen",
-    "--",
-    "--keys=experimental-all",
-    "--locales=full",
-    "--format=blob",
-    "--out=/dev/null",
-    "--overwrite",
-]
+dependencies = ["bakeddata"]
+script_runner = "@duckscript"
+script = '''
+exec --fail-on-error target/release/icu4x-datagen --locales basic moderate --keys experimental-all --format mod --out nonbake
```

Comment:
Clarification: you're not including modern because it's done in config.json?

Reply:
Yes. Each component could select its set independently.